mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -256,6 +256,15 @@ class LogisticRegression @Since("1.2.0") (
   @Since("1.5.0")
   override def getThresholds: Array[Double] = super.getThresholds
 
+  /**
+   * Set suggested depth for treeAggregate or treeReduce (>= 2).
+   * Default is 2.
+   * @group setParam
```

Comment:
This should really be an expert param. Unfortunately, it doesn't seem like `SharedParamsCodeGen` supports that at the moment. We can either hard code these to linear/logistic regression (which isn't a terrible idea since it's hard to predict how reusable this really is) or we can update the code gen file to support expert.